src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -54,7 +54,7 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setAlias('api_platform.naming.resource_path_naming_strategy', $config['naming']['resource_path_naming_strategy']);
+        $container->setAlias('api_platform.operation_path_resolver.default', $config['default_operation_path_resolver']);
```

Comment:
@Ener-Getick:
> It would be weird to decorate a service defined by the user in case he uses its own path resolver, don't you think ?

Isn't this what you're doing here?